fedoracommunity/search/index.py

```diff
@@ -111,51 +117,54 @@ def create_index(self):
     @property
     def latest_release(self):
         if not self._latest_release:
-            response = local.http.get(self.pkgdb_url + "/api/collections")
-            if not bool(response):
-                raise IOError("Unable to find latest release %r" % response)
-            data = response.json()
-
-            # Strip off rawhide
-            data['collections'] = [
-                c for c in data['collections'] if c['version'] != 'devel'
-            ]
+            releases_all = self.get_all_releases_from_bodhi()
```

Comment:
Just a note, not sure if we can/want to do something about it, there is a time after branching where bodhi doesn't know about the new release coming up, so relying on bodhi will make the F27 info not show up until it's activated in bodhi.